ldap/servers/slapd/slap.h

```diff
@@ -281,6 +281,8 @@ typedef void (*VFPV)(); /* takes undefined arguments */
 #define SLAPD_DEFAULT_MAXSIMPLEPAGED_PER_CONN_STR "-1"
 /* We'd like this number to be prime for the hash into the Connection table */
 #define SLAPD_DEFAULT_CONNTABLESIZE 4093 /* connection table size */
+#define SLAPD_DEFAULT_CONNTABLE_NUMLISTS 2 /* connection table lists */
```

Comment:
According to my test there are some drops in throughput/resp_time with any value > 1. I think it should be 1 ATM, until we fix others bottlenecks